### PR TITLE
[EDU-3724] feat: add add filters events

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new page for Add filters on Real-Time Events.
* Modified Real-Time Events reference to remove filter content.

[EDU-3724]: https://aziontech.atlassian.net/browse/EDU-3724?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/add-filters.mdx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: How to add filters on Real-Time Events
+description: Filter your analysis with the specific variables and type of data you want to receive.
+meta_tags: azion, edge, observe, observability, logs, analysis
+namespace: docs_integrate_grafana
```
Comment (hannah-sk) [inline src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/add-filters.mdx:5]:
```suggestion
namespace: docs_add_filters_events
```

File: src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/add-filters.mdx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+description: Filter your analysis with the specific variables and type of data you want to receive.
+meta_tags: azion, edge, observe, observability, logs, analysis
+namespace: docs_add_filters_events
+permalink: /documentation/products/guides/observe/add-filters-events/
+menu_namespace: observeMenu
+---
+
+**Real-Time Events** allows you to filter your analysis with the specific variables and type of data you want to receive.
+
+Once you [understand Real-Time Events logs](/en/documentation/products/guides/observe/understand-logs/) and decide the variables you want to query, you can use the filter and query for more exact results.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/add-filters.mdx:12]: exact or precise results?

File: src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/add-filters.mdx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+If you run a query with blank Filter by field, you’ll get all existing records for the variables of the selected data source available during the selected time filter.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/add-filters.mdx:14]:
```suggestion
If you run a query with a blank **Filter by field**, you’ll get all existing records for the variables of the selected data source available during the selected time filter.
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/add-filters.mdx (hunk 2 above)
Comment (hannah-sk) [inline src/content/docs/en/pages/observe-journey/real-time-events/analyze-logs/add-filters.mdx:12]:
```suggestion
Once you [understand Real-Time Events logs](/en/documentation/products/guides/observe/understand-logs/) and decide the variables you want to query, you can use the filter and query for more accurate results.
```

**Pull request merged**